pyproject.toml

```diff
@@ -54,7 +54,7 @@ strip-ansi = "*"
 pytest-subprocess = "^1.5.2"
 
 [tool.poetry.group.container.dependencies]
-pymupdf = "^1.24.10"
+pymupdf = "1.24.11" # Last version to support python 3.8 (needed for Ubuntu Focal support)
```

Comment:
I've updated this in [`719cb1a` (#971)](https://github.com/freedomofpress/dangerzone/pull/971/commits/719cb1a8ebe004a6b303298bf29864bce2af34b6). It's worth noting that we now can have different conflicting versions in the pyproject.toml file.

I'm saying this because I'm not sure in practice it will be possible to have different versions between container and vendored groups. Poetry will need version that can match all groups requirements.

Reply:
Oh, I'm sorry. I just checked and you're right. Poetry cannot handle conflicting versions (I thought it could). I did a check and got this:

```
Resolving dependencies... (0.0s)

Incompatible constraints in requirements of dangerzone (0.7.1):
PyMuPDF (>=1.23.3,<2.0.0)
pymupdf (>=1.24.12,<2.0.0)
pymupdf (==1.24.11)
```

In that case, your original fix was better, sorry for the extra work :sweat:. I'm approving this PR, feel free to ditch this commit.

_Note that there's a typo in the latest fix ("vendorized" vs "vendored")._

Reply:
Okay, let's revert this then :+1: 

> Note that there's a typo in the latest fix ("vendorized" vs "vendored").

Yep it's fixed in a subsequent fixup commit, thanks!